config/crd/bases/observability.openshift.io_clusterlogforwarders.yaml

```diff
@@ -0,0 +1,51 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.9.2
+  creationTimestamp: null
+  name: clusterlogforwarders.observability.openshift.io
+spec:
+  group: observability.openshift.io
+  names:
+    kind: ClusterLogForwarder
+    listKind: ClusterLogForwarderList
+    plural: clusterlogforwarders
+    singular: clusterlogforwarder
+  scope: Namespaced
```

Comment:
Double checking here. Is `Namespaced` intended?

Reply:
Yes.  Maps to mCLF concept.  Added reference in enhancement based upon comment